iOSTest/AppDelegate.m

```diff
@@ -26,6 +26,15 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     self.window.rootViewController = self.navController;
 
+    // Set the Navigation Bar properties
+    
+    [[UINavigationBar appearance] setBarTintColor:[UIColor colorWithRed:14.0/255.0 green:92.0/255.0 blue:137.0/255.0 alpha:1.0]];
+    
+    [[UINavigationBar appearance] setTintColor: [UIColor whiteColor]];
+    
+    // Change the title color to white
+    [[UINavigationBar appearance] setTitleTextAttributes:@{NSForegroundColorAttributeName:[UIColor whiteColor]}];
```

Comment:
Remove all spaces and remove "Change the title color to white". "Set the Navigation Bar Properties" tells the developer everything they need to know.